src/script/conversation/ConversationRepository.ts

```diff
@@ -197,7 +197,7 @@ export class ConversationRepository {
       if (conversation) {
         // add/remove users from the conversation (if any)
         const missingUserIds = missing.map(({userId}) => userId);
-        const knownUsers = conversation.participating_user_ets().map(user => user.qualifiedId);
+        const knownUsers = conversation.allUserEntities.map(user => user.qualifiedId);
```

Comment:
This was missing the selfUser (and thus would always try to add the selfUser in the conv)